<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>321</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Add User</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonScanId_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scan ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Card ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cardID_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>School ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="schoolID_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="password_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="userType_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>User</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Business</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teller</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Secret:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="secret_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OTP:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="otpt_addUser">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
